<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1326</width>
        <height>674</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QCheckBox" name="enabledCbx">
         <property name="text">
          <string>Enabled</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Type</string>
            </property>
            <property name="buddy">
             <cstring>cloudTypeCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="cloudTypeCmb"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>Remote Directory</string>
            </property>
            <property name="buddy">
             <cstring>remoteDirectoryLed</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLineEdit" name="remoteDirectoryLed"/>
          </item>
          <item row="0" column="7">
           <widget class="QCheckBox" name="createCloudCbx">
            <property name="text">
             <string>Create Cloud Via API</string>
            </property>
           </widget>
          </item>
          <item row="0" column="8" colspan="2">
           <widget class="QPushButton" name="createCloudBtn">
            <property name="text">
             <string>Create Cloud Via Website</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>File Sync Method</string>
            </property>
            <property name="buddy">
             <cstring>fileSyncMethodCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="fileSyncMethodCmb"/>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>OneTrainer Directory</string>
            </property>
            <property name="buddy">
             <cstring>onetrainerDirectoryLed</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="onetrainerDirectoryLed"/>
          </item>
          <item row="1" column="7">
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>Cloud Name</string>
            </property>
            <property name="buddy">
             <cstring>cloudNameLed</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="8" colspan="2">
           <widget class="QLineEdit" name="cloudNameLed"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>API Key</string>
            </property>
            <property name="buddy">
             <cstring>apiKeyLed</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="apiKeyLed"/>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>Huggingface Cache Directory</string>
            </property>
            <property name="buddy">
             <cstring>huggingfaceCacheLed</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLineEdit" name="huggingfaceCacheLed"/>
          </item>
          <item row="2" column="7">
           <widget class="QLabel" name="label_45">
            <property name="text">
             <string>Type</string>
            </property>
            <property name="buddy">
             <cstring>subTypeCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="8" colspan="2">
           <widget class="QComboBox" name="subTypeCmb"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Hostname</string>
            </property>
            <property name="buddy">
             <cstring>hostnameLed</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="hostnameLed"/>
          </item>
          <item row="3" column="4" colspan="2">
           <widget class="QCheckBox" name="installOnetrainerCbx">
            <property name="text">
             <string>Install OneTrainer</string>
            </property>
           </widget>
          </item>
          <item row="3" column="7">
           <widget class="QLabel" name="label_46">
            <property name="text">
             <string>GPU</string>
            </property>
            <property name="buddy">
             <cstring>gpuCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="8">
           <widget class="QComboBox" name="gpuCmb"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="buddy">
             <cstring>portSbx</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QSpinBox" name="portSbx">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>Install Command</string>
            </property>
            <property name="buddy">
             <cstring>installCommandLed</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QLineEdit" name="installCommandLed"/>
          </item>
          <item row="4" column="7">
           <widget class="QLabel" name="label_47">
            <property name="text">
             <string>Volume Size</string>
            </property>
            <property name="buddy">
             <cstring>volumeSizeSbx</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="8" colspan="2">
           <widget class="QSpinBox" name="volumeSizeSbx"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>User</string>
            </property>
            <property name="buddy">
             <cstring>userLed</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLineEdit" name="userLed"/>
          </item>
          <item row="5" column="4" colspan="2">
           <widget class="QCheckBox" name="updateOnetrainerCbx">
            <property name="text">
             <string>Update OneTrainer</string>
            </property>
           </widget>
          </item>
          <item row="5" column="7">
           <widget class="QLabel" name="label_48">
            <property name="text">
             <string>Min Download</string>
            </property>
            <property name="buddy">
             <cstring>minDownloadSbx</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="8" colspan="2">
           <widget class="QSpinBox" name="minDownloadSbx"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>Cloud ID</string>
            </property>
            <property name="buddy">
             <cstring>cloudIdLed</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QLineEdit" name="cloudIdLed"/>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="tensorboardTcpTunnelCbx">
            <property name="text">
             <string>Tensorboard TCP Tunnel</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>221</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="8" column="3">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="8" column="6">
           <spacer name="verticalSpacer_9">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="0">
           <widget class="QCheckBox" name="detachRemoteTrainerCbx">
            <property name="text">
             <string>Detach Remote Trainer</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_49">
            <property name="text">
             <string>Reattach ID</string>
            </property>
            <property name="buddy">
             <cstring>reattachIdLed</cstring>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="reattachIdLed">
            <property name="minimumSize">
             <size>
              <width>156</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QPushButton" name="reattachBtn">
            <property name="text">
             <string>Reattach Now</string>
            </property>
           </widget>
          </item>
          <item row="10" column="5">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>156</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="10" column="7">
           <widget class="QLabel" name="label_50">
            <property name="text">
             <string>Action On Finish</string>
            </property>
            <property name="buddy">
             <cstring>onFinishCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="10" column="8" colspan="2">
           <widget class="QComboBox" name="onFinishCmb"/>
          </item>
          <item row="11" column="0">
           <widget class="QCheckBox" name="downloadSamplesCbx">
            <property name="text">
             <string>Download Samples</string>
            </property>
           </widget>
          </item>
          <item row="11" column="7">
           <widget class="QLabel" name="label_51">
            <property name="text">
             <string>Action On Error</string>
            </property>
            <property name="buddy">
             <cstring>onErrorCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="11" column="8" colspan="2">
           <widget class="QComboBox" name="onErrorCmb"/>
          </item>
          <item row="12" column="0">
           <widget class="QCheckBox" name="downloadOutputModelCbx">
            <property name="text">
             <string>Download Output Model</string>
            </property>
           </widget>
          </item>
          <item row="12" column="7">
           <widget class="QLabel" name="label_52">
            <property name="text">
             <string>Action On Detached</string>
            </property>
            <property name="buddy">
             <cstring>onDetachedCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="12" column="8" colspan="2">
           <widget class="QComboBox" name="onDetachedCmb"/>
          </item>
          <item row="13" column="0">
           <widget class="QCheckBox" name="downloadSavedCheckpointsCbx">
            <property name="text">
             <string>Download Saved Checkpoints</string>
            </property>
           </widget>
          </item>
          <item row="13" column="7">
           <widget class="QLabel" name="label_53">
            <property name="text">
             <string>Action On Detached Error</string>
            </property>
            <property name="buddy">
             <cstring>onDetachedErrorCmb</cstring>
            </property>
           </widget>
          </item>
          <item row="13" column="8" colspan="2">
           <widget class="QComboBox" name="onDetachedErrorCmb"/>
          </item>
          <item row="14" column="0">
           <widget class="QCheckBox" name="downloadBackupsCbx">
            <property name="text">
             <string>Download Backups</string>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QCheckBox" name="downloadTensorboardLogCbx">
            <property name="text">
             <string>Download Tensorboard Logs</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QCheckBox" name="deleteRemoteWorkspaceCbx">
            <property name="text">
             <string>Delete Remote Workspace</string>
            </property>
           </widget>
          </item>
          <item row="17" column="0">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>3</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="9">
           <widget class="QToolButton" name="gpuBtn">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>enabledCbx</tabstop>
  <tabstop>cloudTypeCmb</tabstop>
  <tabstop>fileSyncMethodCmb</tabstop>
  <tabstop>apiKeyLed</tabstop>
  <tabstop>hostnameLed</tabstop>
  <tabstop>portSbx</tabstop>
  <tabstop>userLed</tabstop>
  <tabstop>cloudIdLed</tabstop>
  <tabstop>remoteDirectoryLed</tabstop>
  <tabstop>onetrainerDirectoryLed</tabstop>
  <tabstop>huggingfaceCacheLed</tabstop>
  <tabstop>installOnetrainerCbx</tabstop>
  <tabstop>installCommandLed</tabstop>
  <tabstop>updateOnetrainerCbx</tabstop>
  <tabstop>createCloudCbx</tabstop>
  <tabstop>createCloudBtn</tabstop>
  <tabstop>cloudNameLed</tabstop>
  <tabstop>subTypeCmb</tabstop>
  <tabstop>gpuCmb</tabstop>
  <tabstop>gpuBtn</tabstop>
  <tabstop>volumeSizeSbx</tabstop>
  <tabstop>minDownloadSbx</tabstop>
  <tabstop>tensorboardTcpTunnelCbx</tabstop>
  <tabstop>detachRemoteTrainerCbx</tabstop>
  <tabstop>downloadSamplesCbx</tabstop>
  <tabstop>reattachIdLed</tabstop>
  <tabstop>reattachBtn</tabstop>
  <tabstop>downloadOutputModelCbx</tabstop>
  <tabstop>downloadSavedCheckpointsCbx</tabstop>
  <tabstop>downloadBackupsCbx</tabstop>
  <tabstop>downloadTensorboardLogCbx</tabstop>
  <tabstop>deleteRemoteWorkspaceCbx</tabstop>
  <tabstop>onFinishCmb</tabstop>
  <tabstop>onErrorCmb</tabstop>
  <tabstop>onDetachedCmb</tabstop>
  <tabstop>onDetachedErrorCmb</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>enabledCbx</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
